quick_start_demo/5-min-Mosaic-AI-Agents-Demo-with-Synthetic-Eval.py

```diff
@@ -0,0 +1,699 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Demo overview
+# MAGIC This notebook shows you how to use Mosaic AI to evaluate and improve the quality/cost/latency of a tool-calling Agent, deploying the resulting Agent to a web-based chat UI.
+# MAGIC
+# MAGIC Using Mosiac AI [Agent Evaluation](https://docs.databricks.com/en/generative-ai/agent-evaluation/index.html), [Agent Framework](https://docs.databricks.com/en/generative-ai/agent-framework/build-genai-apps.html), [MLflow](https://docs.databricks.com/en/generative-ai/agent-framework/log-agent.html) and [Model Serving](https://docs.databricks.com/en/generative-ai/agent-framework/deploy-agent.html), we will:
+# MAGIC 1. Generate synthetic evaluation data from a document corpus
+# MAGIC 2. Create a tool-calling Agent with a Retriever tool
+# MAGIC 3. Evaluate the Agent's quality/cost/latency across several foundational models
+# MAGIC 4. Deploy the Agent to a web-based chat app
+# MAGIC
+# MAGIC Requirements: 
+# MAGIC * Use a Serverless or MLR/DBR 14.3+ compute cluster
+# MAGIC * Databricks Serverless & Unity Catalog enabled
+# MAGIC * CREATE MODEL access to a Unity Catalog schema
+# MAGIC * Permission to create Model Serving endpoints
+# MAGIC
+# MAGIC **How to use this notebook**:
+# MAGIC 1. Update the following cell to configure the UC location where the agent model will be register to
+# MAGIC
+# MAGIC  ✅✏️ CONFIGURE THE UC MODEL NAME HERE
+# MAGIC
+# MAGIC 2. Click Run all
+# MAGIC
+# MAGIC ![alt text](https://github.com/databricks/genai-cookbook/blob/demo_gifs/quick_start_demo/demo_gifs/demo_overview.gif?raw=true "Title")
```

Comment:
@peter lu has new gifs for all the gifs throughout the notebook - can we use those?